### ISSUE TITLE
Tools with durability acts strange.

### ISSUE THREAD

**Issue by craft90** [The-Nexus-Project/Limitless#27]
**Describe the bug**
Now this may seem like a mod issue. But considering it happened to two different mods in this modpack I feel like I ought to mention it anyway. For some reason tools with durability (Vanilla Food Pantry Cutter, PickleTweaks grass mesh.) goes away after using it once.

**To Reproduce**
1. Go to a crafting table (bug happened both in vanilla and Silent Gems crafting station)
2. Use a cutter (Vanilla Food Pantry) or grass mesh (PickleTweaks) with charcoal/raw meat etc. or gravel respectively.
3. Notice that once you have done the crafting process, exiting and/or grabbing the tool back will make it go away. If you make several crafts the mesh remains for one but goes away on the next with whatever you were supposed to get.

**Expected behavior**
You would expect to get the tool back minus whatever durability it takes to make the craft.

**Additional details**
I got 8 GB out of 16 GB RAM dedicated to Minecraft.

**Comment by craft90** [comment]
Upon further testing I noticed that the issue does not occur in the 2x2 crafting window, so that works as a workaround. I've not tried it with the Cyclic crafting upgrade though so I can't say if it's just an inventory issue. In the 2x2 it works as intended. Which may hint at it being a shapeless vs shaped crafting issue. But how that may occur I do not know. No script I could find referenced VanillaFoodPantry but I didn't look too closely into it.
In addition I managed to dupe charcoal in my testing. I used the crafting table on a stick to use the cutter to get bit o' heat from charcoal. As expected the cutter went away. However I got the charcoal returned aswell as the bit o' heat.

Also at this stage I got 10 GB dedicated RAM. I upped it thinking it may just be lag.

Also I thought of something else. Silent Gems does allow you to repair tools by using items. So maybe that mod is somehow conflicting?
